### ISSUE TITLE
Artists with an ampersand in their name get split into two different artists

### ISSUE THREAD

**Issue by ephemeraleternity** [krateng/maloja#154]
If I play a song by "The World Is A Beautiful Place & I Am No Longer Afraid To Die", Maloja scrobbles the song as if two separate artists wrote the song: "I Am No Longer Afraid To Die,The World Is A Beautiful Place". 

Relevant piece of info from the log:
`[database] Incoming scrobble [Client: mpdscribble | API: Legacy Audioscrobbler]: {'track_artists': ['The World is a Beautiful Place & I am No Longer Afraid to Die'], 'track_title': 'Trouble', 'scrobble_time': 1655133849}`

And [here's](https://transfer.sh/get/X7XIsL/screenshot.png) a screenshot of the relevant scrobble on the Maloja web interface.

**Comment by FoxxMD** [comment]
You can use [rules](https://github.com/krateng/maloja#customization) to define how Maloja should handle special cases with artists. If you check out `rules.info` it even has an example for grouping artists with an ampersand together.

Make a new file in your maloja config/data directory like `maloja/rules/world.tsv` with the contents:

```
belongtogether	The World Is A Beautiful Place & I Am No Longer Afraid To Die
```

Then restart maloja and try scrobbling the track again.

**Comment by ephemeraleternity** [comment]
Oh! Oh no! I should've read the documentation. Sorry about this and thank you for your help!

**Comment by copey02** [comment]
Hello - 
I am running into similar issue, with the same artist actually. 
as well as artist Akron/Family. 
Maloja is splitting these into 2 artists.

For the life of me, I can't figure out how to get my custom rules working and I can't tell if it's because Maloja is not picking up my rules file, or if the rules themselves are incorrect. 

I have a .tsv file in /data/rules 
`[copey@fedora rules]$ pwd
/home/copey/Documents/docker/maloja/rules
[copey@fedora rules]$ ls
customrules.tsv  krateng_artistsingroups.tsv  krateng_dach.tsv  krateng_specialsymbols.tsv  predefined  rules.info
[copey@fedora rules]$ `

contents of my customrules.tsv file is just:
`belongtogether  The World Is A Beautiful Place & I Am No Longer Afraid To Die
belongtogether  Akron/Family`

I've restarted maloja using `maloja restart` command

everytime I rescrobble a song by one of these artists, there is no change, they are stil split into 2. 
it seems as though Maloja is not picking up my rules.
Is there any way to validate the server is seeing the rules? 

As a test, I moved my rule file to the 'predefined' foler, and it was picked up in the GUI and I was able to "Add" it as a predefined ruleset, but still no changes were made when scrobbling.
